src-java/swmanager-topology/swmanager-storm-topology/src/main/java/org/openkilda/wfm/topology/switchmanager/bolt/SwitchManagerHub.java

```diff
@@ -143,6 +146,8 @@ public void init() {
                 carrier -> new SwitchRuleService(carrier, persistenceManager.getRepositoryFactory()));
         createLagPortService = registerService(
                 serviceRegistry, "lag-create", this, carrier -> new CreateLagPortService(carrier, config));
```

Comment:
please constants

Reply:
Can't agree. I do not want to produce garbage. One time use "constant" do not deserve own name and "global" definition.